src/components/Modal/index.js

```diff
@@ -12,54 +12,55 @@ const MODAL_ANIMATION_DURATION = getCSSMilisecsValue(
 
 const Wrapper = ({ children }) => wrapWithClass(style.inner, children)
 
-class Modal extends Component {
-  static defaultProps = {
-    shouldCloseOnOverlayClick: true,
-  }
+const Modal = () => {
+  const {
+    translate,
+    isFullScreen,
+    containerId,
+    containerEl,
+    shouldCloseOnOverlayClick,
```

Comment:
I think we still need to set the default value of `true` here for `shouldCloseOnOverlayClick` unless I've missed something.

Reply:
This should be out-of-scope for this PR. Actually we need a stronger system for these kinds of stuff, either:
- Declare component's static attributes propTypes & defaultProps, with validations from prop-types, or
- Exhaust Props & State typing in current type system, like Flowtype or TypeScript
which are [noted here](https://wiki.onfido.net/display/SDK/Development+guideline+for+Web+SDK#DevelopmentguidelineforWebSDK-c.Readability&Maintainability)

Reply:
I agree about it being out-of-scope for the PR. But I checked it out locally & it looks like there is code that relies on this prop always having a defined value as there is a TypeError thrown if SDK is configured with `useModal` set to `true`.

Reply:
Ah you mean the `defaultProps`? The static declaration has been moved down a bit ⬇

Reply:
Yeah, I meant the `defaultProps` stuff. 😄